go.mod

```diff
@@ -5,15 +5,16 @@ go 1.12
 require (
 	github.com/SkyAPM/go2sky v0.3.1-0.20200329092408-8b3e4d536d8d
 	github.com/TarsCloud/TarsGo v0.0.0-20181112071624-2d42457f2025
+	github.com/alibaba/sentinel-golang v0.2.1-0.20200509115140-6d505e23ef30
 	github.com/apache/dubbo-go-hessian2 v1.5.0
 	github.com/beorn7/perks v1.0.1 // indirect
 	github.com/c2h5oh/datasize v0.0.0-20171227191756-4eba002a5eae
-	github.com/envoyproxy/go-control-plane v0.6.9
+	github.com/envoyproxy/go-control-plane v0.8.0
```

Comment:
fixme: It's already used spec version of `go-control-plane` to replace.
`replace github.com/envoyproxy/go-control-plane => github.com/envoyproxy/go-control-plane v0.6.9`